docs/getting-started.md

```diff
@@ -1,54 +1,49 @@
 ---
 slug: getting-started
 append_help_link: true
-description: "Start by running Semgrep locally to scan your code. It runs offline on uncompiled code: no code leaves your machine."
+description: "How to start using Semgrep CLI."
```

Comment:
What’s the reasoning behind this change? My sense is the previous description added some more details beyond what’s directly in or implied by the URL and title themselves. This description seems mostly reflexive of the title.

Reply:
I am not sure that the previous text is an actual description, it is the first sentence of the article actually... I can expand the description, I just do not like to see the first sentence of the article reused as a description. @pabloest

Reply:
Fair enough! Then I suggest something different. Otherwise it’s a swap of one duplicate thing (the first sentence of the article) with another duplicate thing (the page title).

Reply:
@pabloest ok, pushed this, check this description out:

`description: "This getting started guides you through the installation of Semgrep, shows you how to run Semgrep locally, and gives an overview of benefits which you may reap by writing Semgrep rules and by using Semgrep CI."`